src/index.js

```diff
@@ -1,6 +1,29 @@
-export default () => {
-  const greeting = 'Hello from new repository';
-  console.log(greeting);
+import axios from 'axios';
+import fsp from 'fs/promises';
+import path from 'path';
 
-  return greeting;
+const fetchPageData = (url) => axios.get(url)
+  .then((response) => response.data)
+  .catch((error) => {
+    throw error;
+  });
+
+const getFileNameFromUrl = (url) => {
+  const [, urlWithoutScheme] = url.split('://');
```

Comment:
```suggestion
const urlWithoutProtocol = url.replace('https://', '')
```

можно проще

Reply:
```suggestion
  const urlWithoutScheme = url.replace(/https?:\/\//, '');
```

тогда лучше через регулярку, чтобы `http` тоже попадал